Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -183,4 +268,36 @@ extension BrowserViewController {
             }
         }
     }
+    
+    func shareTrackersAndAdsWithEmail(_ count: Int) {
+        benchmarkNotificationPresented = false
+
+        dismiss(animated: true) {
+            // TODO: Share with Email
```

Comment:
Are we waiting on product/design decision on sharing implementations? Perhaps may want to switch to WIP/Draft PR

Reply:
Yes this is waiting for product/design to finish final sharing part. 
I added information in the description about it and marked it as Blocked